ui/DesignSystem/Component/Transaction.svelte

```diff
@@ -36,6 +37,7 @@
   const updateAvatar = async () => (avatar = await subject.avatarSource);
 
   $: updateAvatar();
+  $: payer = formatPayerFromIdentity($session.data.identity);
```

Comment:
The payer should be passed in, having a component in the DesignSystem depend on a store is backwards of what we have been working towards. Esp. as the data can be undefined depending on the state of the remote store.